README.md

```diff
@@ -1,6 +1,7 @@
 Pa11y Runner
 ===
-`pa11y_runner` is a simple wrapper for running pa11y scans against local copies of websites.
+`pa11y_runner` is a simple wrapper for the [pa11y](https://pa11y.org/) CLI tool for runnign scans against local copies of websites.
+
 
 Currently CSV reports are generated for WCAG2A and WCAG2AA standards using both axe and htmlcs runners.
```

Comment:
Would be good to add links to these tools as well.

Reply:
Done